Doc/library/ipaddress.rst

```diff
@@ -217,16 +217,20 @@ write code that handles both IP versions correctly.  Address objects are
       :RFC:`4291` for details.  For example,
       ``"0000:0000:0000:0000:0000:0abc:0007:0def"`` can be compressed to
       ``"::abc:7:def"``.
+
+      Optionally, the string may also have a scope (zone) id, expressed
+      with a suffix ``%<scope_id>``. If present, the scope id must be non-empty,
+      and may not contain ``%``.
```

Comment:
So, in conclusion, the section should be changed to:

```
      Optionally, the string may also have a scope (zone) ID, expressed
      with a suffix ``%<scope_id>``. If present, the scope ID must be non-empty,
      and may not contain ``%``.
```

I agree with the suggestion, but it can sometimes be easier for the author when the recommended changes are explicitly mentioned.

Reply:
> it can sometimes be easier for the author when the recommended changes are explicitly mentioned.

True indeed! :slightly_smiling_face: See here:
https://github.com/python/cpython/pull/13772#discussion_r313085202

I'd already suggested text for this, which is why this comment focused on explaining my reasoning about a difference between my suggested text and the version subsequently added to the PR.

Reply:
Ah, I see. It can be quite difficult to keep track of all of the comments in a PR, especially once the total number of them starts to get high. (: